### PR TITLE
Maint-26510: Feature New Composer : Create a new folder icon is not functional

### PR DESCRIPTION
**Pull request by Rdidmohamedelhadi**
Problem : when create new folder in drives view, we cannot show `icon create a new folder` because we need to show only the drives, so the fix is we will hide the `icon create new folder` in the driver's view.

### CODE REVIEW

File: ecm-wcm-extension/src/main/webapp/attachments-selector/components/ExoFoldersFilesSelector.vue [hunk 1]
```diff
@@ -39,7 +39,7 @@
       <div :class="showSearchInput? 'visible' : ''" class="selectorActions">
         <input id="searchServerAttachments" ref="searchServerAttachments" v-model="searchFilesFolders" type="text" class="searchInput">
         <a :class="showSearchInput ? 'uiIconCloseServerAttachments' : 'uiIconFilter'" class="uiIconLightGray" @click="showSearchDocumentInput()"></a>
-        <a v-if="modeFolderSelectionForFile || modeFolderSelection" :title="$t('attachments.filesFoldersSelector.button.addNewFOlder.tooltip')" rel="tooltip" data-placement="bottom" class="uiIconLightGray uiIconAddFolder" @click="addNewFolder()"></a>
+        <a v-if="(modeFolderSelectionForFile || modeFolderSelection) && currentDrive.title" :title="$t('attachments.filesFoldersSelector.button.addNewFOlder.tooltip')" rel="tooltip" data-placement="bottom" class="uiIconLightGray uiIconAddFolder" @click="addNewFolder()"></a>
```
Comment (ahamdi) [inline ecm-wcm-extension/src/main/webapp/attachments-selector/components/ExoFoldersFilesSelector.vue:42]: Couldn't we just test on the currentDrive variable ?

Comment (Rdidmohamedelhadi) [reply ecm-wcm-extension/src/main/webapp/attachments-selector/components/ExoFoldersFilesSelector.vue:42]: we just use the check used in 
https://github.com/exodev/ecms/blob/f3063ad6e6279b3d41b2f95fe1c13b784a053c7e/ecm-wcm-extension/src/main/webapp/attachments-selector/components/ExoFoldersFilesSelector.vue#L58

Comment (ahamdi) [reply ecm-wcm-extension/src/main/webapp/attachments-selector/components/ExoFoldersFilesSelector.vue:42]: I see, you initialize the object and is not Undefined by default. You need to check everytime if an attribute is empty .
It is better if you initialize it to null or empty object without specifying the variables and do the check on the Object currentDrive itself.

Comment (Rdidmohamedelhadi) [reply ecm-wcm-extension/src/main/webapp/attachments-selector/components/ExoFoldersFilesSelector.vue]: done !

Comment (ahamdi) [reply ecm-wcm-extension/src/main/webapp/attachments-selector/components/ExoFoldersFilesSelector.vue]: Could you simplify more and initialize the object to null

Comment (Rdidmohamedelhadi) [reply ecm-wcm-extension/src/main/webapp/attachments-selector/components/ExoFoldersFilesSelector.vue]: ok

File: ecm-wcm-extension/src/main/webapp/attachments-selector/components/ExoFoldersFilesSelector.vue [hunk 2]
```diff
@@ -10,9 +10,9 @@
           <p class="documents" data-toggle="tooltip" rel="tooltip" data-placement="bottom"
              data-original-title="Documents">{{ $t('attachments.drawer.drives') }}</p>
         </div>
-        <div v-if="currentDrive.title" class="currentDrive" @click="openDrive(currentDrive)">
+        <div v-if="currentDrive" class="currentDrive" @click="openDrive(currentDrive)">
           <span class="uiIconArrowRight"></span>
-          <a :title="currentDrive.title" :class="currentDrive.isSelected? 'active' : ''" class="currentDriveTitle" data-toggle="tooltip" rel="tooltip"
```
Comment (ahamdi) [inline ecm-wcm-extension/src/main/webapp/attachments-selector/components/ExoFoldersFilesSelector.vue:15]: This is unneded I think

Comment (Rdidmohamedelhadi) [reply ecm-wcm-extension/src/main/webapp/attachments-selector/components/ExoFoldersFilesSelector.vue:15]: done

**Pull request merged**